docs/releasenotes/unreleased/other.1.rst

```diff
@@ -0,0 +1,13 @@
+Disabler keyword are now consistent with Robotidy (#933)
+--------------------------------------------------------
+
+Disabling the line from the linting can be done using disabler comment::
+
+    # robocop:disable=rule_name
+
+Robotidy also supports disablers but using different keywords::
+
+    # robotidy:off
+
+We have decided to make it more consistent and use Robotidy ``on`` and ``off`` markers instead of ``enable`` and
```

Comment:
```suggestion
We have decided to make it more consistent and use ``on`` and ``off`` markers (mimicking [Robotidy tool](https://github.com/MarketSquare/robotframework-tidy)) instead of ``enable`` and
```